monailabel/utils/others/generic.py

```diff
@@ -216,3 +219,72 @@ def is_openslide_supported(name):
     if ext and ext in supported_ext:
         return True
     return False
+
+
+def get_bundle(app_dir, conf):
+    MONAI_ZOO_INFO = "https://raw.githubusercontent.com/Project-MONAI/model-zoo/dev/models/model_info.json"
+    MONAI_ZOO_SOURCE = "github"
+    MONAI_ZOO_REPO = "Project-MONAI/model-zoo/hosting_storage_v1"
+
+    model_dir = os.path.join(app_dir, "model")
+
+    zoo_info = requests.get(conf.get("zoo_info", MONAI_ZOO_INFO)).json()
+    zoo_source = conf.get("zoo_source", MONAI_ZOO_SOURCE)
+    zoo_repo = conf.get("zoo_repo", MONAI_ZOO_REPO)
+
+    available = {k.replace(".zip", ""): v for k, v in zoo_info.items()}
+    models = conf.get("bundles")
+    if not models:
+        print("")
+        print("---------------------------------------------------------------------------------------")
+        print("Provide --conf bundles <name>")
+        print("Following are the available bundles.  You can pass comma (,) separated names to pass multiple")
+        print("   -c bundles {}".format("\n    -c bundles ".join(available.keys())))
+        print("---------------------------------------------------------------------------------------")
+        print("")
+        exit(-1)
+
+    models = models.split(",")
+    models = [m.strip() for m in models]
+    # First check whether the bundle model directory is in model-zoo, if no, check local bundle directory.
+    # Use zoo bundle if both exist
+    invalid_zoo = [m for m in models if m != "all" and not available.get(m)]
+    invalid = [m for m in invalid_zoo if not os.path.isdir(os.path.join(model_dir, m))]
+
+    # Exit if model is not in zoo and local directory
+    if invalid:
+        print("")
+        print("---------------------------------------------------------------------------------------")
+        print(f"Invalid Model(s) are provided: {invalid}")
+        print("Following are the available models.  You can pass comma (,) separated names to pass multiple")
+        print("    -c models all\n    -c models {}".format("\n    -c models ".join(available.keys())))
+        print("Or provide valid local bundle directories")
+        print("---------------------------------------------------------------------------------------")
+        print("")
+        exit(-1)
+
+    # bundle models
+    bundles: Dict[str, str] = {}
+    for n in models:
+        # Load from local if any bundle is not in Zoo
```

Comment:
lets combine this and https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/monaibundle/main.py#L44-L84 into some common reusable function..

Reply:
Thanks, combined and made common, it can be used for radiology/monaibundle apps.